libraries/AP_ExternalAHRS/AP_ExternalAHRS_VectorNav.cpp

```diff
@@ -760,6 +771,11 @@ bool AP_ExternalAHRS_VectorNav::pre_arm_check(char *failure_msg, uint8_t failure
     return true;
 }
 
+uint8_t AP_ExternalAHRS_VectorNav::num_gps_sensors(void) const
+{
+    return type == TYPE::VN_INS ? 1 + has_dual_gnss : 0;
```

Comment:
Diving into this I'm a little confused on the issue. I don't think we are publishing GPS2 data to AP inside our driver anywhere, but I have it documented [here](https://ardupilot.org/plane/docs/common-external-ahrs-vectornav.html#use-as-an-external-sensor-set) to set that param for the VN-300. Should we be publishing GPS2 data somewhere? Or should we remove that param as one to be set, and make it so num_gps_sensors is either 0 or 1?